app/views/layouts/application.html.erb

```diff
@@ -1,37 +1,31 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title><%= raw page_title %></title>
-  <link href='https://fonts.googleapis.com/css?family=Nixie+One|Open+Sans:400,600,700' rel='stylesheet' type='text/css'>
-  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.2/css/bootstrap.min.css">
-
-  <%= stylesheet_link_tag    'application', media: 'all'%>
-  <%= javascript_include_tag 'application'%>
-  <%= csrf_meta_tags %>
-
-  <script src="//netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
-  <script src="//use.fontawesome.com/16a020f0b8.js"></script>
-
-  <!-- Check device width for responsive media queries -->
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-  <%= favicon_link_tag "favicon.png" %>
-
-</head>
-<body>
-<div class="container">
-  <div class="shover"></div> <!-- shover -->
-
-  <p id="notice"><%= notice %></p>
-  <p id="alert"><%= alert %></p>
-
-  <%= render "layouts/header" %>
-
-  <%= render "shared/overlay_modal" %>
-
-  <%= yield %>
-
-</div> <!-- container -->
-
-</body>
+  <head>
+    <title><%= raw page_title %></title>
+    <link href='https://fonts.googleapis.com/css?family=Nixie+One|Open+Sans:400,600,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.2/css/bootstrap.min.css">
+
+    <%= stylesheet_link_tag    'application', media: 'all'%>
+    <%= javascript_include_tag 'application'%>
+    <%= csrf_meta_tags %>
+
+    <script src="//netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
+    <script src="//use.fontawesome.com/16a020f0b8.js"></script>
+
+    <!-- Check device width for responsive media queries -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <%= favicon_link_tag "favicon.png" %>
+
+  </head>
+  <body>
+    <%= render "layouts/environment_banner" %>
+    <%= render "layouts/header" %>
+    <div class="container">
+      <p id="notice"><%= notice %></p>
+      <p id="alert"><%= alert %></p>
+      <%= render "shared/overlay_modal" %>
+      <%= yield %>
+    </div>
+  </body>
```

Comment:
The changes to this file are mostly in indention. But in this section, I did rearrange the divs a little bit to make the navbar smarter and allow space for the env banner.